config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   resources :line_items
   resources :carts
+  get '/my_cart', to: 'carts#index', as: "my_cart"
```

Comment:
This would be better directed to `'carts#show'`, as per my earlier message. It would necessitate either moving it above the `resources :carts` declaration, or passing the methods you want resources to be built for. (`resources :carts, only: [:new, :create, etc]`)

Reply:
Again, agree with what you're saying, but I want to leave it for the moment to see if the book will start changing routes and accounting for access to different carts when it comes to authentication later on